<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Vivek-PC" ignored="2" name="Test" tests="1" failures="1" timestamp="06 Jun 2021 05:35:18 GMT" time="22.822" errors="0">
  <testcase name="invalidUserName" time="9.642" classname="com.qa.tests.LoginTests">
    <failure type="java.lang.AssertionError" message="expected [Username and password do not match any user in this services.] but found [Username and password do not match any user in this service.]">
      <![CDATA[java.lang.AssertionError: expected [Username and password do not match any user in this services.] but found [Username and password do not match any user in this service.]
at com.qa.tests.LoginTests.invalidUserName(LoginTests.java:40)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- invalidUserName -->
  <testcase name="successFulLogin" time="0.0" classname="com.qa.tests.LoginTests">
    <ignored/>
  </testcase> <!-- successFulLogin -->
  <testcase name="InvalidPassword" time="0.0" classname="com.qa.tests.LoginTests">
    <ignored/>
  </testcase> <!-- InvalidPassword -->
</testsuite> <!-- Test -->
